Application/Features/WordFeatures/Commands/PostWordCommand.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Application.Interfaces;
+using Domain.Entites;
+using MassTransit;
+using MediatR;
+using SharedModelsLibrary;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Application.Features.WordFeatures.Commands
+{
+    public class PostWordCommand : IRequest<Word>
+    {
+        public Word _word;
+        public PostWordCommand(Word word)
+        {
+            _word = word;
+        }
+        public class PostWordCommandHandler : IRequestHandler<PostWordCommand, Word>
+        {
+            private readonly IWordsDbContext _context;
+            private readonly IPublishEndpoint _publishEndpoint;
+            public PostWordCommandHandler(IWordsDbContext context, IPublishEndpoint publishEndpoint)
+            {
+                _context = context;
+                _publishEndpoint = publishEndpoint;
+            }
+            private static DateTime DateToday
+            {
+                get { return DateTime.Today.ToUniversalTime(); }
```

Comment:
Let's move it to separate service e.g. DateTimeService. It can help you with mocking in unit tests, and also in the future if you will need to synchronizing this DateTime between scaled microservices, it would be easier to do

Reply:
done.